form_app/lib/src/sign_in_http.dart

```diff
@@ -54,16 +68,19 @@ class _SignInHttpDemoState extends State<SignInHttpDemo> {
               children: [
                 ...[
                   TextFormField(
+                    textInputAction: TextInputAction.next,
```

Comment:
Similar to the above, I'd recommend adding focus nodes to all the fields in this and the other forms.